### PR TITLE
Create main.tf

### PR DESCRIPTION
**Pull request by IlonaFursova99**

### CODE REVIEW

File: hw9/main.tf [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  source_image_reference {
+    publisher = "Canonical"
+    offer     = "0001-com-ubuntu-server-jammy"
+    sku       = "22_04-lts"
+    version   = "latest"
+  }
+  #Path to public key
+  admin_ssh_key {
+    username   = "azureuser"
+    public_key = file("C:\\Users\\ilona\\OneDrive\\Рабочий стол\\Work\\DevOps course\\hw9 terraform\\sshkey.pub")
```
Comment (tony-apr) [inline hw9/main.tf]: this path is not accessible to me

File: hw9/main.tf [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    destination_port_range     = "80"
+    source_address_prefixes    = ["85.198.144.37", "10.95.0.116"] #Prituln server address + my local IP
+    destination_address_prefix = "*"
+  }
+
+}
+
+# Virtual Machines 1,2
+
+resource "azurerm_linux_virtual_machine" "vm" {
```
Comment (tony-apr) [inline hw9/main.tf:101]: created VMs don't belong to the azurerm_network_security_group, pls check "azurerm_subnet_network_security_group_association" resource documentation

File: hw9/main.tf (hunk 1 above)
Comment (IlonaFursova99) [reply hw9/main.tf]: Should be changed to yours or I can insert my own

File: hw9/main.tf (hunk 2 above)
Comment (IlonaFursova99) [reply hw9/main.tf:101]: Resolved in the latest commit

**Comment by IlonaFursova99** [comment]
Results:
image
image

**Pull request merged**